### ISSUE TITLE
Any plans to make new release?

### ISSUE THREAD

**Issue by kloczek** [Tencent/rapidjson#1881]
I think that it would be good to flush currently committed changes and make new release :)

**Comment by jacgrie** [comment]
Yes, that would also help us a lot!

**Comment by dcooley** [comment]
Duplicate of #1862, #1820, #1774, #1006

**Comment by smhdfdl** [comment]
I agree, but I would like any new release to include fixes for the following issues that do not yet have committed PRs.
- #1843 (PR #1848) 
- #1841 (code written, not in PR yet)
- #1892 (code written, not in PR yet)

**Comment by MrPointer** [comment]
@smhdfdl With all due respect, I think it can wait until the next release afterward, assuming it won't take 5 more years 😅  
Seriously though, we just need a new, official release ASAP, containing whatever there is in **master** now, as it seems like this project stopped being maintained...

**Comment by Livius90** [comment]
Use nlohmann C++ json lib, it is much better in releasing and documentation. I gave up waiting for new rapidjson release, i am using this other from now.
https://github.com/nlohmann/json

**Comment by kloczek** [comment]
Gentle ping .. any update? 🤔 
In git repo already is accumulated +500 commits since last release.
